<?xml version="1.0"?>
<launch>

  <arg name="gui" default="true"/>

  <param name="SpeedGain" value="3"/>

  <include file="$(find challenge_project)/launch/gazebo.launch">
  <arg name="x" value="0"/>
  <arg name="y" value="5.25"/>
  <arg name="z" value="0.0175"/>
  <arg name="roll" value="0"/>
  <arg name="pitch" value="0"/>
  <arg name="yaw" value="3.1415"/>
  <arg name="world" value="$(find challenge_project)/worlds/challenge1.world"/>
  <arg name="gui" value="$(arg gui)"/>
  </include>

  <node pkg="challenge_project" name="line_detection" type="line_following.py" required="true"/>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/camera/image_raw" />
  </node>

</launch>
